azure_functions_worker/bindings/meta.py

```diff
@@ -111,6 +111,14 @@ def get_datum(binding: str, obj: typing.Any,
     return datum
 
 
+def is_cache_supported(datum: datumdef.Datum):
```

Comment:
Could you please clarify here that we are checking if the caching a datatype is supported or not. More like `does_datatype_support_caching`? As I think this confuses with `is_function_data_cache_enabled` and only looking at the definition of this function specifies why this is required.

Also, could you please make it private?

Reply:
Also - why not look at `in`

```python
>>> x = ("string", "bytes")
>>> a = "bytes"
>>> b = "blah"
>>> a in x
True
>>> b in x
False
```

Reply:
Accepted your suggested change.